.github/workflows/tests_linux_cpp.yml

```
@@ -48,11 +48,12 @@ jobs:
         pl_backend: ["lightning_qubit"]
         enable_kernel_omp: ["OFF", "ON"]
         enable_kernel_avx_streaming: ["OFF", "ON"]
+        enable_lapack: ["OFF", "ON"]
         exclude:
         - enable_kernel_omp: OFF
           enable_kernel_avx_streaming: ON
     timeout-minutes: 60
-    name: C++ Tests (${{ matrix.pl_backend }}, ENABLE_KERNEL_OMP=${{ matrix.enable_kernel_omp }}, ENABLE_KERNEL_AVX_STREAMING=${{ matrix.enable_kernel_avx_streaming }})
+    name: C++ Tests (${{ matrix.pl_backend }}, ENABLE_KERNEL_OMP=${{ matrix.enable_kernel_omp }}, ENABLE_KERNEL_AVX_STREAMING=${{ matrix.enable_kernel_avx_streaming }}), ENABLE_LAPACK=${{ matrix.enable_lapack }})
```

Not related to your PR, but just noticed that we can make this consistent with other names too,
```suggestion
    name: C++ Tests (${{ matrix.pl_backend }}, enable_kernel_omp=${{ matrix.enable_kernel_omp }}, enable_kernel_avx_streaming=${{ matrix.enable_kernel_avx_streaming }}), enable_lapack=${{ matrix.enable_lapack }})
```